lib/ecto/changeset.ex

```diff
@@ -2307,14 +2307,14 @@ defmodule Ecto.Changeset do
   In order to use the check constraint, the first step is
   to define the check constraint in a migration:
 
-      create constraint("users", :price_must_be_positive, check: "price > 0")
+      create constraint("users", :price, :price_must_be_positive, check: "price > 0")
```

Comment:
this was correct for _migrations_, could you revert this change? Good catch on the changeset function tho!

Reply:
ups, thanks for catching that! I fixed it in 5df8138